plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -231,30 +236,42 @@ export class PersonState {
         }
     }
 
-    private updatedPersonProperties(personProperties: Properties): Properties {
-        const updatedProperties = { ...personProperties }
+    private updatedPersonProperties(person: Partial<Person>): Partial<Person> {
+        const updatedPerson = {
+            properties: { ...person.properties } || {},
+            // ignore timestamp and operation only updates as we don't really use them yet
```

Comment:
This comment does not make sense - what is being ignored below this line?

Reply:
we're not making a copy here and that works fine because we're not using op and timestamp anywhere, if we were using them somewhere we'd need to make a copy to make sure we don't change the person we passed in here, like we need to do above for properties.